src/common/meta/src/ddl/alter_table/region_request.rs

```diff
@@ -91,6 +92,23 @@ fn create_proto_alter_kind(
                 add_columns,
             })))
         }
+        Kind::ChangeColumnTypes(x) => {
+            let change_column_types = x
+                .change_column_types
+                .iter()
+                .map(|change_column_type| ChangeColumnType {
+                    column_name: change_column_type.column_name.clone(),
+                    target_type: change_column_type.target_type,
+                    target_type_extension: change_column_type.target_type_extension.clone(),
+                })
+                .collect();
+
+            Ok(Some(alter_request::Kind::ChangeColumnTypes(
+                ChangeColumnTypes {
+                    change_column_types,
+                },
+            )))
```

Comment:
Could we pass `x` directly? They should have the same type now.
```suggestion
        Kind::ChangeColumnTypes(x) => {
            Ok(Some(alter_request::Kind::ChangeColumnTypes(x)))
```